.gitignore

```diff
@@ -44,6 +44,9 @@ install
 # IDE
 .idea
 
+#macOS
+.DS_Store
```

Comment:
I still can see .DS_Store in config directory

Reply:
fixed